### DISCUSSION TITLE
How to have GLPI Agent limited installation on windows ?

### DISCUSSION THREAD

**Discussion by musklor** [glpi-project/glpi-agent#276] [Q&A] [answered]
Hi, I need to install a glpi agent in order to launch/use only inventory (no deployment).

ADDLOCAL=feat_COLLECT is the right parameter for the MSI installation?

**Comment by g-bougard** [comment, marked as answer]
No this will also install Collect task.
To only install inventory task (which is always installed by default), just use `ADDLOCAL=feat_AGENT`.
I just updated the documentation to clarify this value can be used.
